### DISCUSSION TITLE
Panel component

### DISCUSSION THREAD

**Discussion by awcodes** [filamentphp/filament#2305] [Ideas]
@danharrin 

Here’s the panel component I built that we talked about on discord the other day.

You mentioned possibly replacing the section or card component with it. It supports collapsible, description, etc.

Let me know your thoughts.

Cheers.

![0FCC8531-4D10-4D4A-A774-143B6D28CB5D](https://user-images.githubusercontent.com/3596800/167221291-bdc901ae-41d4-4772-bc53-a11ff504b493.png)

**Comment by danharrin** [comment]
Hey! This looks perfect.

Ideally, we'd replace the Section component view with this, preserving all available options. We should then discard the current Card component and extend Section instead.

Thoughts?

**Reply by awcodes** [reply]
Replacing the section view sounds good.

personally I’d be hesitant to drop the card in case there’s a use case to just have an area that always shows as just an informational view. But I guess that could just be the base panel view where it’s styling is opted into with the methods that are chained onto it. But might be overkill.

I could see it working either way.
